py_mmd_tools/nc_to_mmd.py

```diff
@@ -955,6 +955,45 @@ def get_geographic_extent_rectangle(self, mmd_element, ncin):
 
         return data
 
+    def get_operational_status(self, mmd_element, ncin):
+        """ Get the operational_status from the processing_level ACDD
+        attribute.
+        """
+        VALID = [
+            "Operational",
+            "Pre-Operational",
+            "Experimental",
+            "Scientific",
+            "Not available"
+        ]
+        repetition_allowed = mmd_element.pop('maxOccurs', '') not in ['0', '1']
+        if repetition_allowed:
+            raise ValueError("This is not expected...")
+        acdd = mmd_element["acdd"]
+        acdd_key = list(acdd.keys())[0]
+        if acdd_key in ncin.ncattrs():
+            ostatus = ncin.getncattr(acdd_key)
+        else:
+            ostatus = "Not available"
+
+        operational_status = ""
+        valid_statuses = [v.lower() for v in VALID]
+        if ostatus.lower() not in valid_statuses:
+            self.missing_attributes['errors'].append(
+                "The ACDD attribute 'operational_status' must "
+                "follow a controlled vocabulary from MMD (see "
+                "https://htmlpreview.github.io/?https://github."
+                "com/metno/mmd/blob/master/doc/mmd-specification."
+                "html#operational-status).")
+        else:
+            # Need to make a new list of lists to use the filter
+            # function for comparison between ostatus and valid
+            # operational_status'es from the controlled vocabulary
+            xx = [[ostatus, tt] for tt in VALID]
+            x = filter(lambda a: a[0].lower() == a[1].lower(), xx)
```

Comment:
Hva er hensikten her?
``` python
if ostatus in valid_statuses: 
    operational_status = ostatus
```

Reply:
ostatus kan være både uppercase og lowercase. Vil godta begge, men legge inn rett i MMD. Blir for pirkete å be brukere skrive "Not available" fremfor "not available".